aries-site/src/pages/templates/forms.mdx

```diff
@@ -135,37 +104,84 @@ All Form input elements need to be contained within a FormField.
   <ChangePasswordExample />
 </Example>
 
-## Settings
+### Customize
 
 <Example
-  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/forms/SettingsExample.js"
-  figma="https://www.figma.com/file/3fkwBelW5UsCbfwdDCJkT8/HPE-Form-Templates?node-id=54%3A459"
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/forms/CustomizeExample.js"
+  figma="https://www.figma.com/file/3fkwBelW5UsCbfwdDCJkT8/HPE-Form-Templates?node-id=54%3A608"
 >
-  <SettingsExample />
+  <CustomizeExample />
 </Example>
 
-## Pay
+## Guidance with validation
+
+validation should happen both on formfield level as well as submission of the form. 
+
+### Required vs. optional fields
+
+When possible, restrict Forms only to fields that are required. This ensures that the Form remains concise for the user. However, there may be cases where optional fields may be fitting.
+
+When using a mixture of required and optional fields, apply the "required" prop to any given FormField label in which will disable form submission until this field has been completed.
+
+For standard forms, use an asterisk(*) to indicate required field/controls. 
+Optional fields receive no treatment.
+
+Sign In and Sign Up forms do not use an asterisk for required fields. All other standard forms/long forms will use an asterisk for required fields. 
+
+Optional fields receive no treatments.
+
+
+### Using help & info text
+
+In most cases, the FormField label should be clear enough to instruct the user how to complete a field. However, if help text is required, ensure that it is kept brief and precise.
+
+Help/Info text may be relevant for cases such as [password requirements](#sign-up) or [additional resources](#customize) on a field's context
+
+### Field validation
+
+In most cases field validation is used in order to ensure that the data filled out passes any set paramaters. 
+
+Field validation should be followed with an error message that will help the user understand the problem and will be able to fix it. 
+
+For any required field the following should be used "This is a required field"
+
+Form validation will occur before any api call is made to ensure the correct information is being used.
+
+### Form submission
+
+Visual feedback is important to instill confidence in a user after form submission.
+
+**On submission, your form should:**
+
+- In the case of an error, focus should be placed on the input element where the error occurred.
+- Provide user with indication that form was submitted successfully (e.g. notification, success page, etc.)
+- Display all relevant errors accompanied by text explaining actions needed to resolve.
+- Allow user to continue navigating through the application immediately
+
+Never force the user to wait for post-submission processing to complete before they can continue using an application.
+
+## Variants
```

Comment:
Wondering if this should be called "Examples of Validation" -- something that's more descriptive of why these are grouped together